source/_integrations/xiaomi_aqara.markdown

```diff
@@ -23,7 +23,12 @@ ha_integration_type: integration
 
 The **Xiaomi Gateway (Aqara)** {% term integration %} allows you to integrate [Xiaomi](https://www.mi.com/en/) Aqara-compatible devices into Home Assistant.
 
-Please note, there are two versions of the hub: v1 and v2. v1 can be used with Home Assistant without any problems, however, v2 might be less straight forward when it comes to enabling the local API, and might even require you to open up your device in order to do so. Xiaomi has suggested this is in the pipeline.
+{% note %}
+There are two versions of the hub: v1 and v2.
+
+- v1 can be used with Home Assistant without any problems.
+- v2 might be less straight forward when it comes to enabling the local API. It might even require you to open up your device in order to do so. Xiaomi has suggested this is in the pipeline. If you have a Hub 2 and are experiencing issues, checkout the [troubleshooting](#troubleshooting) section.
```

Comment:
```suggestion
- v2 might be less straightforward when it comes to enabling the local API. It might even require you to open up your device to do so. Xiaomi has suggested this is in the pipeline. If you have a Hub 2 and are experiencing issues, check out the [troubleshooting](#troubleshooting) section.
```